htdocs/admin/emailcollector_list.php

```diff
@@ -594,6 +596,33 @@
 print '</table>'."\n";
 print '</div>'."\n";
 
+$formcategory = new FormCategory($db);
+
+print load_fiche_titre($langs->trans("Other"), '', '');
+print '<table class="noborder centpercent">';
+
+print '<tr class="liste_titre">';
+print '<td>'.$langs->trans("Parameter").'</td>';
+print '<td></td>';
+print '<td></td>';
+print "</tr>\n";
+
+// Hide e-mail headers from collected messages
+print '<tr class="oddeven"><td>'.$langs->trans("EmailCollectorHideMailHeaders").'</td>';
+print '<td class="left">';
+if ($conf->use_javascript_ajax) {
+	print ajax_constantonoff('MAIN_EMAILCOLLECTOR_MAIL_WITHOUT_HEADER');
+} else {
+	$arrval = array('0' => $langs->trans("No"), '1' => $langs->trans("Yes"));
+	print $formcategory->selectarray("MAIN_EMAILCOLLECTOR_MAIL_WITHOUT_HEADER", $arrval, $conf->global->TICKET_AUTO_READ_WHEN_CREATED_FROM_BACKEND);
```

Comment:
You should use 
$form->selectarray(

Reply:
Fixed in https://github.com/Dolibarr/dolibarr/pull/20669/commits/bdfe7f12d90051e004dc2c9a715d0a7aeb0868c7